executor/python/server.py

```diff
@@ -41,10 +42,11 @@ def createLockFile():
     path = "/tmp/.lock"
     open(path, "w")
     os.chmod(path, 0o660)
-    print('Writing lock-file to: '+ path)
+    logging.info('Writing lock-file to: '+ path)
 
 def serve():
     fxwatcherPort = os.getenv('WATCHER_PORT', '50051')
+    fxmeshPort = os.getenv('MESH_PORT', '50052')
```

Comment:
앞에 fx를 붙이는게 괜찮은가 아닌가는 잘 모르겠네요. ㅎ

```suggestion
    fxwatcher_port = os.getenv('WATCHER_PORT', '50051')
    fxmesh_port = os.getenv('MESH_PORT', '50052')
```